module/bdev/rbd/bdev_rbd.c

```diff
@@ -26,6 +26,7 @@ static int bdev_rbd_count = 0;
 struct bdev_rbd_pool_ctx {
 	rados_t *cluster_p;
 	char *name;
+	char *rbd_namespace_name;
```

Comment:
The namespace is really a RADOS concept, not RBD.  I'd suggest just `namespace_name`:
```suggestion
	char *namespace_name;
```
There shouldn't be any confusion with NVMe namespaces because this is an RBD bdev specific struct.

Reply:
@idryomov , Hi thank you, 
I will change it to be with rados_ prefix

Reply:
I don't insist, but I'm still of the opinion that just `namespace_name` would be sufficient across the PR.  All structs and methods (including the RPC bits) that you being touched here are specific to RBD.  All of this is at level of SPDK bdev -- NVMe namespaces are pretty far removed from this code.  Just like `pool_name` isn't named `rados_pool_name`, there is no need to go out of the way and say `rados_namespace_name`.

Reply:
@idryomov there is a convention in SPDK to keep the parameter name identical as it passes from one layer to the next. As we call the function from within the gateway, where the name "namespace" means something else, it's best to keep the parameter name "rados_namespace_name". The internal field can be named differently but it would make our lives a little more difficult when chasing after the variable while debugging. It's not a crucial issue but so is the "rados" prefix. We do have already a "rados_ctx" in the SPDK code so it wouldn't be the first variable to use a "rados" prefix.

Reply:
I doubt that e.g. `user_id` is a unique name across all of SPDK.

Also, I don't buy the idea of aligning parameter names across the entire stack and having some "freedom" only with internal variable names because this doesn't scale.  The fact that the gateway calls something shouldn't have any bearing on SPDK, particularly with regard to naming -- the gateway and SPDK are two separate projects each with its own API, living in different repositories, after all.

That said, this is just a naming nit, so feel free to ignore.  Naming is one of the hardest problems in computer science ;)